microsoft-edge/progressive-web-apps-chromium/how-to/offline.md

```diff
@@ -11,9 +11,11 @@ keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft S
 ---
 # Offline and network connectivity support in Progressive Web Apps
 
-For many years organizations were reluctant to invest heavily in web-based software over native software because web applications depended on stable network connections. Today, the web platform now offers robust options that enable users to continue working, even if the network connection becomes unstable or goes completely offline.
+For many years, organizations were reluctant to invest heavily in web-based software over native software, because web applications depended on stable network connections.  Today, the web platform offers robust options that enable users to continue working even if the network connection becomes unstable or goes offline.
```

Comment:
The Progressive Web Apps (PWA) platform offers robust options that enable users to continue working even if the network connection becomes unstable or goes offline.
```